content/guides/core/registry/aliases.md

```diff
@@ -0,0 +1,149 @@
+
+# Link the artifact version to the collection
+# Add one or more aliases to this artifact version
+run.link_artifact(
+    artifact = artifact, 
+    target_path = target_path, 
+    aliases = ["<alias_1>", "<alias_2>"]
+    )
+```
+{{% /tab %}}
+{{< /tabpane >}}
+
+### Protected aliases
+Use a [protected alias]({{< relref "aliases/#protected-aliases" >}}) to both label and identify artifact versions that should not be modified or deleted. For example, consider using a `production` protected alias to label and identify artifact versions that are in used in your organization's machine learning production pipeline.
+
+Only [registry admins]({{< relref "/guides/core/registry/configure_registry/#registry-roles" >}}) can create protected aliases and add or remove protected aliases from an artifact version. Users (Member and Viewer roles) can not delete a collection that contains a version with a protected alias, and they also cannot unlink it from the registry. See [Configure registry access]({{< relref "/guides/core/registry/configure_registry.md" >}}) for information on how to manage users and assign roles in a registry.
+
+<!-- Admins can only create protected aliases in the W&B Registry UI. Once created, an admin can add a protected alias to an artifact version with the W&B Registry UI or the Python SDK. -->
+
+<!-- Protected aliases are not shared across registries.  -->
+
+Common protected aliases include:
+
+- **Production**: The artifact version is ready for production use.
+- **Staging**: The artifact version is ready for testing.
+
+#### Create protected aliases
+
+The following steps describe how to create a protected alias in the W&B Registry UI:
+
+1. Navigate to the Registry App.
+2. Select a registry.
+3. Click the gear button on the top right of the page to view the registry's settings.
+4. Within the **Protected Aliases** section, click on the plus icon (**+**) to add one or more protected aliases.
+
+If successful, you will see one or more protected aliases appear as a gray rectangle with a lock icon in the **Protected Aliases** section. 
+
+{{% alert %}}
+Unlike custom non protected aliases, protected aliases can only be created in the W&B Registry UI and not programmatically with the Python SDK. Once created, admins can add a protected alias to an artifact version with the W&B Registry UI or the Python SDK.
+{{% /alert %}}
+
+The following steps describe how to add a protected alias to an artifact version with the W&B Registry UI:
+
+1. Navigate to the W&B Registry.
+2. Click the **View details** button in a collection.
+3. Within the **Versions** section, select the **View** button for a specific artifact version.
+4. Select the plus icon (**+**) to add one or more protected aliases next to the **Aliases** field.
+
+Admins can add a protected alias to an artifact version programmatically with the Python SDK. Make sure that the protected alias already exists in the registry.
```

Comment:
Only one admin can add a protected alias, right? There are a few other instances of subject/object count disagreement.

Reply:
A registry can have multiple admins.

Reply:
Yeah understood, but only one admin at a time can add a given alias at a time. So maybe "Any admin can"? I can see either way, tbh!